README.md

```diff
@@ -41,7 +40,8 @@ Jump to competencies for a particular level:
 
 ### Community & Citizenship
 
-- You do things to benefit your peers and community such as participating in meetings and providing constructive feedback on ideas and initiatives.
+- You do things to benefit your peers and community such as participating in meetings, including providing constructive feedback on ideas and initiatives.
+- You advocate for yourself and others when something is unclear or you believe there is an alternate approach worth discussion.
```

Comment:
I would reframe the "alternate approach" part of this to be its own line, and phrase as "You ask questions about approaches to solving problems and suggest alternate options"; "worth discussion" is IMO too subjective a statement to be useful